src/modules/data/shared/move/modal/index.js

```diff
@@ -0,0 +1 @@
+export { default } from './container';
```

Comment:
would it make sense to move this to elements or RSVP and ticket blocks instead, as the data folder is for state management?

Reply:
Yes I'll do that